README.md

```diff
@@ -24,86 +41,28 @@ This project contains:
-
-</details>
-
 ## Documentation
 
 - A brief user manual and developer docs are hosted on our [GitHub Pages](https://ostis-ai.github.io/sc-machine).
+  - <details>
+      <summary>Build documentation locally</summary>
+
+    ```sh
+    pip3 install mkdocs mkdocs-material
+    mkdocs serve
+    # and open http://127.0.0.1:8000/ in your browser
+    ```
+    </details>
 - Full documentation, including:
 
   - core concepts
   - rationale behind the sc-machine
   - system design
   - software interfaces
 
-  is redistributed in a form of the [SCn-TeX document](https://github.com/ostis-ai/ostis-web-platform/blob/develop/docs/main.pdf).
-
-  or **alternatively** you can build sc-machine documentation only. To do that refer to the [scn-latex-plugin](https://github.com/ostis-ai/scn-latex-plugin) documentation.
-
-<details>
-  <summary>Build documentation locally</summary>
-
-```sh
-# make sure you're using python12
-pip3 install mkdocs mkdocs-material
-mkdocs serve
-# and open http://127.0.0.1:8000/ in your browser
-```
-</details>
-
-## Quick start
-
-Semantic network storage is a core of any ostis-system, so you can use a reference system named [OSTIS Platform](https://github.com/ostis-ai/ostis-web-platform) to get it up and running quickly.
-
-## Installation
-
-- Docker:
-  We provide a Docker image for this project. Head to [Installing with Docker](https://ostis-ai.github.io/sc-machine/docker) to learn more. It's the recommended way to deploy the sc-machine.
-- Native:
-  If you do not have the option to deploy the system using Docker, please refer to the docs:
-  [Build system](https://ostis-ai.github.io/sc-machine/build/build_system/)
-
-  Note: currently, the sc-machine isn't _natively_ supported on Windows.
-
-## Usage
-
-- Docker
-
-  ```sh
-  # create empty KB sources folder
-  mkdir kb && cat "." > kb/repo.path
-  # note: at this stage you can move your KB sources to the ./kb folder
-  
-  # build kb
-  docker compose run --rm machine build
-  # run sc-machine
-  docker compose up
-  ```
-
-- Native
-
-  Note: Currently, only Linux (Ubuntu-20.04, Ubuntu-22.04, Ubuntu-24.04) and macOS are supported by this installation method. If you're going to use it, it might take a while to download dependencies and compile the components. Use it only if you know what you're doing!
-
-  ```sh
-  # build kb
-  ./build/<Debug|Release>/bin/sc-builder -c ./sc-machine.ini -i <path to kb folder with SCs and SCg sources (or path to repo.path file)> -o <output path> --clear
-  # launch sc-machine
-  ./build/<Debug|Release>/bin/sc-machine -c ./sc-machine.ini
-  ```
-
-Most of these commands have a help page bundled inside, so if you have any questions or want to customize the command behavior, use `--help` flag to learn more about them.
-
-## Config
+  is redistributed in a form of the SCn-TeX document [compiled in PDF format](https://github.com/ostis-ai/ostis-web-platform/blob/develop/docs/main.pdf).
```

Comment:
I'm not sure this is an alternative. One documentation has something that the other one doesn't and vice versa. I suggest referring to this documentation as containing more rigorous theoretical calculations